cypress/integration/edit_segment.spec.js

```diff
@@ -0,0 +1,384 @@
+/// <reference types="cypress" />
+// Tests the ability to edit a segment
+
+describe("Edit segment", () => {
+    beforeEach(() => {
+    
+        cy.visit(Cypress.env('login_url'));
+        cy.get("[name=csrfmiddlewaretoken]")
+          .should("exist")
+          .should("have.attr", "value")
+          .as("csrfToken");
+    
+        cy.get("@csrfToken").then((token) => {
+          cy.request({
+            method: "POST",
+            url: Cypress.env("login_url"), 
+            form: true,
+            body: {
+              username: "cypress",
+              password: "1234asdf",
+            },
+            headers: {
+              "X-CSRFTOKEN": token,
+            },
+          });
+        });
+    })
+
+    it("shows original word", () => {
+        
+        cy.visit(Cypress.env('home'));
+
+        cy.get(".table")
+            .should("be.visible");
+
+        cy.get('a[name="word-link"]:first')
+            .then((word) => {
+                const w = word[0].id
```

Comment:
Is there a better name for this than `w`?

Reply:
Yes.